test/e2e/dra/README.md

```diff
@@ -13,11 +13,30 @@ for the test driver and that the e2e test has full control over all gRPC calls,
 in case that it needs that for operations like error injection or checking
 calls.
 
-# Cluster setup
+## Cluster setup
 
 The container runtime must support CDI. The latest cri-o releases contain
 support, containerd 1.6.x does not. To bring up a kind cluster with containerd
 built from their main branch, use:
 
-    test/e2e/dra/kind-build-image.sh dra/node:latest && \
+
+```bash
+$ test/e2e/dra/kind-build-image.sh dra/node:latest && \
```

Comment:
This step can fail on Mac with this error:
```
Starting to build Kubernetes
ERROR: This script requires a minimum bash version of 4.2, but got version of 3.2
```
Unless newer bash version is installed with `brew install bash`

Reply:
Thanks @bart0sh , I have updated the PR to include it. I am wondering if this a requirement for macOS users that should be mentioned "more globally" like in the dev environment setup given the number of scripts used by make rules

Reply:
yes, that makes sense to me. `make ginkgo` also fails with the same error. It would be nice to refer some k/k documentation regarding dev setup if it exists.

Reply:
I think all scripts used by make rules fail when run with a too old bash version. There is a [section](https://github.com/kubernetes/community/blob/master/contributors/devel/development.md#bash-version-requirement) in the k/k doc about it. I removed specific guidelines for macOS, I assume even linux users will face this issue if they use a bash version < 4.2. With your PR and the small context in this one, both linux and macOS users should be able to run DRA e2e tests with kind w/o issue